docs/terraform.md

```diff
@@ -0,0 +1,19 @@
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| domain | Domain that should be resolved special way (point to var.ip address) | string | - | yes |
+| flavor | Flavor depends of OS and init system | string | `debian-systemd` | no |
+| ip | IP for domain | list | `<list>` | no |
+| name |  | string | - | yes |
```

Comment:
Get the name, namespace, stage descriptions from one of the other modules.

Reply:
done